pkg/syncer/spec/spec_process_test.go

```diff
@@ -494,7 +494,7 @@ func TestSyncerProcess(t *testing.T) {
 					}),
 				deployment("theDeployment", "test", "root:org:ws", map[string]string{
 					"state.internal.workload.kcp.dev/us-west1": "Sync",
-				}, nil, nil),
+				}, nil, []string{"workload.kcp.dev/syncer-us-west1"}),
```

Comment:
Could you add: " - syncer finalizer already there" in the test description ?

Reply:
done